hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -44,7 +44,13 @@
 import org.apache.hadoop.ozone.om.response.OMClientResponse;
 import org.apache.hadoop.ozone.om.response.bucket.OMBucketCreateResponse;
 import org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos;
-import org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos.*;
```

Comment:
I see this exist in master. Could you please avoid this change in this JIRA? Let's do this in master when we get chance, rather than doing here in this branch unrelated way. ( It could create unnecessary conflicts some times while merging)

Reply:
Sure.

Reply:
I have fired a JIRA HDDS-6329 to address all the star imports.